### PR TITLE
Add protobuf 2.6.1: 32bit and 64bit

### PR DESCRIPTION
**Pull request by alarcher**
Contribution from Predrag Zecevic as prerequirement of mosh (https://www.illumos.org/issues/5195).

### CODE REVIEW

**Comment by pyhalov** [comment]
Why library is 32-bit only?

**Comment by alarcher** [comment]
It seems 64bit support on SunOS compatible needs more work than packaging.
Do you want to wait to have 64bit working before merging ?

**Comment by pyhalov** [comment]
It would be better. However, if it's too complicated, we can postpone it.

**Comment by alarcher** [comment]
2.6.1 has improved Solaris support.

File: components/protobuf/Makefile [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+# CDDL HEADER END
+#
+# Copyright (c) 2015, Predrag Zecevic. All rights reserved.
+# Copyright (c) 2015, Aurelien Larcher. All rights reserved.
+#
+include ../../make-rules/shared-macros.mk
+
+COMPONENT_NAME=		protobuf
+COMPONENT_VERSION=	2.6.1
+COMPONENT_REVISION=	1
```
Comment (pyhalov) [inline components/protobuf/Makefile]: Please, remove COMPONENT_REVISION

File: components/protobuf/protobuf.p5m [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+file path=usr/lib/$(MACH64)/libprotoc.so.9.0.1
+file path=usr/lib/$(MACH64)/pkgconfig/protobuf-lite.pc
+file path=usr/lib/$(MACH64)/pkgconfig/protobuf.pc
+link path=usr/lib/libprotobuf-lite.so target=libprotobuf-lite.so.9.0.1
+link path=usr/lib/libprotobuf-lite.so.9 target=libprotobuf-lite.so.9.0.1
+file path=usr/lib/libprotobuf-lite.so.9.0.0
+file path=usr/lib/libprotobuf-lite.so.9.0.1
+link path=usr/lib/libprotobuf.so target=libprotobuf.so.9.0.1
+link path=usr/lib/libprotobuf.so.9 target=libprotobuf.so.9.0.1
+file path=usr/lib/libprotobuf.so.9.0.0
```
Comment (pyhalov) [inline components/protobuf/protobuf.p5m]: Couldn't find 'usr/lib/libprotoc.so.9.0.0' in any of the specified search directories.... Please, gmake clean and update manifest

Comment (alarcher) [inline components/protobuf/protobuf.p5m]: Erf right... my bad.

**Comment by alarcher** [comment]
Updated.

**Pull request merged**